Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +92,33 @@ namespace Terrain
                         ->Attribute(AZ::Edit::Attributes::ReadOnly, &MeshConfiguration::IsClodDisabled)
                     ;
 
+                editContext->Class<ClipmapConfiguration>("Clipmap", "Settings related to clipmap rendering")
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxResolution,
+                        "Macro clipmap max resolution (in texels per meter)",
+                        "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 0.5f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 16.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 128.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution,
+                        "Detail clipmap max resolution (in texels per meter)",
```

Comment:
Suggestion:
"Macro clipmap max texels/m"
"Detail clipmap max texels/m"
It's smaller, and at least to _me_ it's equally understandable... :)